gobblin-example/src/main/java/gobblin/example/wikipedia/WikipediaExtractor.java

```diff
@@ -140,6 +144,24 @@ public WikipediaExtractor(WorkUnitState workUnitState) throws IOException {
     this.reader = new WikiResponseReader();
   }
 
+  private String readProp(String key, WorkUnitState workUnitState) {
```

Comment:
Yes, this is being tracked here: https://github.com/linkedin/gobblin/issues/1022 
More folks have run into it. Its high priority, we will fix it soon.